modules/core/keeper/msg_server_test.go

```diff
@@ -779,6 +779,86 @@ func (suite *KeeperTestSuite) TestUpgradeClient() {
 	}
 }
 
+func (suite *KeeperTestSuite) TestRecoverClient() {
+	var (
+		subject, substitute string
+		subjectClientState  exported.ClientState
```

Comment:
are these variables needed to be defined here? doesn't look like they're mutated in `malleate()`

Reply:
nice catch, yea can be axed from the var block